layout/js/map.js

```diff
@@ -64,6 +65,8 @@ function hovertext(text, evt){
 }
 
 function cursorPoint(evt){  
+  svg = document.querySelector("svg");
```

Comment:
init function is `$(document).ready`, I think we should probably move it to top of this.